### DISCUSSION TITLE
When enabling `namespace_packages` in mypy, `Security` causes `Module not callable` error

### DISCUSSION THREAD

**Discussion by vjpr** [fastapi/fastapi#9050] [Questions]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

```py
from fastapi import APIRouter, Depends, Security

@app.get('/hi')
async def foo(
    id: UUID = Header(None),
    user: User = Security(
        get_current_user_from_token, scopes=[OAuthScopes.READ]
    ),
)
```

_mypy.ini_

```
[mypy]
plugins = pydantic.mypy
namespace_packages = True
```

```
$ dmypy run

Module not callable
```

Interesting thing is that it doesn't error on usage of `Header`...

---

It seems the that `__init.py__` of `import fastapi` is correctly importing `Security` as a function.

```
from .applications import FastAPI
from .datastructures import UploadFile
from .exceptions import HTTPException
from .param_functions import (
    Body,
    Cookie,
    Depends,
    File,
    Form,
    Header,
    Path,
    Query,
    Security, # <--
)
from .routing import APIRouter
```

**Comment by YuriiMotov** [comment]
Can't reproduce. It was probably fixed somewhere
